guide/messaging-gateway-service/messaging_gateway.bal

```diff
@@ -0,0 +1,115 @@
+    host: "localhost",
+    port: 5672,
+    queueName: "Calibration"
+};
+documentation { Attributes associated with the service endpoint. }
+endpoint http:Listener sensorEventListner {
+    port: 9090
+};
+documentation {  via HTTP/1.1. }
+@http:ServiceConfig {
+    basePath: "/service"
+}
+service<http:Service> SensorEventService bind sensorEventListner {
+    @http:ResourceConfig {
+        methods: ["POST"], consumes: ["application/json"], produces: ["application/json"],
+        path: "/activity"
+    }
+    documentation { Resources for reciving geo activities captured from the sensor }
+    activity(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        string msg = requestMessage.toString();
+        mb:Message message = check queuesensorGeoMessage.createTextMessage(msg);
+        _ = queuesensorGeoMessage->send(message);
+        res.setTextPayload("status: Sucessfull");
+        _ = conn->respond(res);
+        log:printInfo("Message recived at /activity: " + msg + "status:forwarded");
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/health"
+    }
+    documentation { Resource for checking the health of the EP of the sensor}
+    health(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        string msg = requestMessage.toString();
+        mb:Message message = check queueHealthCheck.createTextMessage(msg);
+        _ = queueHealthCheck->send(message);
+        res.setTextPayload("status: Sucessfull");
+        _ = conn->respond(res);
+        log:printInfo("Message recived at /health: " + msg + "status:forwarded");
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/maintenance"
+    }
+    documentation { Resource for invoking a service when the sensor needed mainteinance }
+    maintanance(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        string msg = requestMessage.toString();
+        mb:Message message = check queueMaintenance.createTextMessage(msg);
+        _ = queueMaintenance->send(message);
+        res.setTextPayload("status: Sucessfull");
+        _ = conn->respond(res);
+        log:printInfo("Message recived at /maintenance: " + msg + "status:forwarded");
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/calibrate"
+    }
+    documentation { Resource for invoking a service when the sensor needed calibration }
+    calibrate(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        string msg = requestMessage.toString();
+        mb:Message message = check queueCalibration.createTextMessage(msg);
+        _ = queueCalibration->send(message);
+        res.setTextPayload("status: Sucessfull");
```

Comment:
Sucessfull -> successful

Reply:
Fixed the typo Sucessfull to successful.